README.md

```diff
@@ -11,13 +13,82 @@ This tells you if your jobs are done. And stuff like that.
  - Gets input / output dataset names
  - Works with piping, other unix nice things
 
-How I do dat?
--------------
+## Installation
 
- 1. Clone me
- 2. Add the directory to your `PATH`
- 3. Type `pandamon [user.<your user name>]`
- 4. **Pretty colors!!!**
+### Install from PyPI
+
+You can install [`pandamonium` from PyPI][pandamonium_PyPI] into any Python
+virtual environment by simply running
+
+```
+python -m pip install pandamonium
+```
+
+### Install the old style of global scripts
+
+If you want to install the original version of `pandamonium` before it became a
+library and was a set of global level Python scripts you can still do that.
+
+1. Clone the repository at tag [`v0.1`][tag_v0.1]
```

Comment:
I think this is still out of date. It will work with master as well, won't it?

Reply:
It is true that it will work with `master` but as that could possibly be deprecated in the future I wanted to add instructions that will for sure work.